models/hh_psc_alpha_gap.cpp

```diff
@@ -619,15 +619,16 @@ nest::hh_psc_alpha_gap::handle( DataLoggingRequest& e )
 }
 
 void
-nest::hh_psc_alpha_gap::handle( GapJEvent& e )
+nest::hh_psc_alpha_gap::handle( GapJunctionEvent& e )
 {
 
   B_.sumj_g_ij_ += e.get_weight();
 
   size_t i = 0;
-  for ( CoeffArrayIterator it = e.begin(); it != e.end(); ++it )
+  std::vector< uint_t >::iterator it = e.begin();
+  while ( it != e.end() )
   {
-    B_.interpolation_coefficients[ i ] += e.get_weight() * ( *it );
+    B_.interpolation_coefficients[ i ] += e.get_weight() * e.get_coeffvalue( it );
```

Comment:
In this loop, it is not obvious at all how `it` is advanced. This happens as a side-effect of `e.get_coeffvalue( it )`, but to understand that one needs to read that function. Maybe this needs to be like this, but then you ought to add a comment explaining how `it` is updated.